### PR TITLE
[develop] Trello-135Xa1L6: Pass recover API sends email with redirect URL

### PR DESCRIPTION
**Pull request by AndreySurzhan**
[Trello ticket](https://trello.com/c/135Xa1L6)

Wiki page is updated as well
https://github.com/wildeastengineer/MKeepAPI/wiki/Password-recovery

### CODE REVIEW

File: routes/user.js [hunk 1]
```diff
@@ -59,19 +60,12 @@ var userRegisterRoutes = function (router, authenticate, sendError) {
...
-                res.json(result);
-            })
-            .fail(function (error) {
-                sendError(error, res);
-            });
-    });
     
     router.post(recoverPasswordPath, function (req, res) {
-        passRecovery.removePassRecoveryToken(req.query.token)
+        passRecovery.isPasswordRecoveryTokenExisting(req.query.token)
```
Comment (AndreySurzhan) [inline routes/user.js]: @wildeastengineer note please that POST request for recovering password should have password recovery token and username in request query parameters and new password in body. Are you agreed on that format

Comment (wildeastengineer) [inline routes/user.js]: @AndreySurzhan lets put all in body

**Comment by AndreySurzhan** [comment]
@wildeastengineer PR is updated, Wiki is also updated, could you please review

**Comment by wildeastengineer** [comment]
:+1:

**Pull request merged**